pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1410,10 +1410,14 @@ self: super: {
   # https://github.com/haskell/haskell-language-server/issues/611
   haskell-language-server = dontCheck super.haskell-language-server;
 
-  # 2021-01-20
-  # apply-refact 0.9.0.0 get's a build error with hls-hlint-plugin 0.8.0
-  # https://github.com/haskell/haskell-language-server/issues/1240
-  apply-refact = super.apply-refact_0_8_2_1;
+  # 2021-02-08: Jailbreaking because of
+  # https://github.com/haskell/haskell-language-server/issues/1329
+  hls-tactics-plugin = doJailbreak super.hls-tactics-plugin;
+
+  # 2021-02-08: Overrides because nightly is to old for hls 0.9.0
+  lsp-test = doDistribute (dontCheck self.lsp-test_0_11_0_7);
+  haskell-lsp = doDistribute self.haskell-lsp_0_23_0_0;
+  haskell-lsp-types = doDistribute self.haskell-lsp-types_0_23_0_0;
```

Comment:
I'm surprised these get marked as `hydraPlatforms = none;`.  Do you have any idea why this would be happening?

Reply:
`extra-packages` always do. We could have a debate about this, but I think it makes sense, at least when we consider that this rule applies for every extra package version we get when something is newer on hackage than on stackage.

Reply:
Oh, huh, I didn't realize this!